server.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 3000;
+
+const errorHandler = (err, req, res, next) => {
+  console.log(err);
```

Comment:
It would be beneficial to log the error stack trace in addition to the error message for easier debugging. [medium]